script.js

```diff
@@ -0,0 +1,149 @@
+const headerNavbar = document.getElementById('navbarSupportedContent');
+const resultBlock = document.getElementById('results');
+const pages = document.querySelector('.pagination');
+const searchButton = document.getElementById('search');
+const searchInput = document.querySelector('.form-control');
+const loader = document.querySelector('.loader');
+let currentPage = 1;
+const maxInfoLinesCount = 5;
+const API_URL = 'https://swapi.dev/api/';
+let searchMode = false;
+const LocalStorageKeys = {
+    category: 'category'
+}
+const selectedCategory = localStorage.getItem(LocalStorageKeys.category) || CATEGORIES.people;
+
+document.addEventListener("DOMContentLoaded", () => {
+    headerNavbar.querySelector(`#${selectedCategory}`).checked = true;
+    getData(selectedCategory).then((data) => {
+        renderCards(data, selectedCategory);
+        addPagination(data.count);
+    })
+});
+
+
+function getData(path) {
+    resultBlock.innerHTML = null;
```

Comment:
place in separate place and cal before `getData`
rename `resultBlock`

Reply:
renamed to cardsContainer.
Created function clearPage that looks 
```suggestion
function clearPage(){
    pages.innerHTML = null;
    cardsContainer.innerHTML = null;
}
```
and call her before getData